tests/test_on_data/test_imaging_interfaces.py

```diff
@@ -71,6 +75,136 @@ def check_extracted_metadata(self, metadata: dict):
         )
 
 
+class TestScanImageSinglePlaneMultiFileImagingInterface(ScanImageSinglePlaneMultiFileImagingInterfaceMixin, TestCase):
+    data_interface_cls = ScanImageSinglePlaneMultiFileImagingInterface
+    interface_kwargs = [
```

Comment:
`interface_kwargs` should be parameterized by `parameterized.parameterize_class` -- see [discussion on slack](https://catalystneuro.slack.com/archives/C01FAC3F345/p1712161546819629)